pages/cloudflare/howtos/env-vars.mdx

```diff
@@ -0,0 +1,35 @@
+## Environment variables
+
+This entry describe the most sensible way to handle your environment variables which works well both during local development and once your application is deployed to Cloudflare Workers.
+
+On the Cloudflare platform, your environment variables can be stored in either ["Enviroment variables"](https://developers.cloudflare.com/workers/configuration/environment-variables/) or ["Secrets"](https://developers.cloudflare.com/workers/configuration/secrets/). The difference being that Secrets are write only and can not be read back from either the dashboard or the CLI.
```

Comment:
nit: i think the write-only part is already covered by saying that they cannot be read back, and i found it a bit confusing reading that until i realised it was only referring to how they're created and not how they're consumed.

```suggestion
On the Cloudflare platform, your environment variables can be stored in either ["Enviroment variables"](https://developers.cloudflare.com/workers/configuration/environment-variables/) or ["Secrets"](https://developers.cloudflare.com/workers/configuration/secrets/). The difference being that Secrets can not be read back from either the dashboard or the CLI after being created.
```